testing/howard-bc/APKBUILD

```diff
@@ -2,28 +2,29 @@
 # Maintainer: Gavin D. Howard <yzena.tech@gmail.com>
 pkgname=howard-bc
 _pkgname=bc
-pkgver=0.5
-pkgrel=1
+pkgver=1.1.2
+pkgrel=0
 pkgdesc="POSIX bc with GNU extensions"
 url="https://github.com/gavinhoward/bc"
 arch="all"
-license="0BSD"
-depends="!bc"
-source="$pkgname-$pkgver.tar.gz::https://github.com/gavinhoward/$_pkgname/archive/$pkgver.tar.gz"
+license="BSD-2-Clause"
+source="$pkgname-$pkgver.tar.xz::https://github.com/gavinhoward/$_pkgname/releases/download/$pkgver/$_pkgname-$pkgver.tar.xz"
 builddir="$srcdir/$_pkgname-$pkgver"
-# The test suite uses a previously installed version of bc,
-# so it would not make much sense to run it.
-options="!check"
 
 build() {
 	cd "$builddir"
-	CFLAGS="-flto" make release
+	PREFIX=/usr DESTDIR="$pkgdir" EXECSUFFIX=-howard ./configure.sh -GO3
+	make
+}
+
+test() {
```

Comment:
it should be `check()`

Reply:
Okay.

Reply:
use `abuild checksum` to update checksum

Reply:
I have been doing that. Am I supposed to do that *and* commit the change?

Reply:
yep, checksums are required to check downloaded data